tests/virt/node/gpu/vgpu/conftest.py

```
@@ -76,3 +89,163 @@ def hco_cr_with_node_specific_mdev_permitted_hostdevices(
+    yield
+    apply_node_labels(nodes=nodes_with_supported_mig_gpus, labels={"nvidia.com/vgpu.config.state": None})
+    try:
+        next(label_gen)
+    except StopIteration:
+        pass
+
+
+@pytest.fixture(scope="class")
+def non_existent_mdev_bus_mig_nodes(
+    workers_utility_pods,
+    mig_gpu_nodes_labeled_with_vm_vgpu,
```

Ruff (0.15.10)

[warning] Unused function argument: `mig_gpu_nodes_labeled_with_vm_vgpu`

(ARG001)